wazo_call_logd/database/models.py

```diff
@@ -108,6 +118,44 @@ class CallLog(Base):
     )
 
 
+@generic_repr
+class RemoteParty(Base):
+
+    __tablename__ = 'call_logd_remote_party'
+
+    uuid = Column(
+        String(36), server_default=text('uuid_generate_v4()'), primary_key=True
+    )
+    call_log_id = Column(
+        Integer,
+        ForeignKey(
+            'call_logd_call_log.id',
+            name='call_logd_remote_party_call_log_id_fkey',
+            ondelete='CASCADE',
+        ),
+    )
+    kind = Column(
+        Enum(
+            'user',
+            'conference',
+            'meeting',
+            'unknown',
+            name='call_logd_remote_party_kind',
+        ),
+        nullable=False,
+    )
+    description = Column(MutableDict.as_mutable(HSTORE))
```

Comment:
should we indicate nullable=False ?
The description is empty {} or contains the id+name.

Reply:
the description by default is `empty`; now we changed the design.